ui/appui-react/src/appui-react/widget-panels/Content.tsx

```diff
@@ -51,12 +60,37 @@ export function WidgetContent() {
     widget?.initialConfig && isProviderItem(widget?.initialConfig)
       ? widget?.initialConfig.providerId
       : undefined;
+
+  const layoutStore = useLayoutStore();
+  const stateRef = React.useRef(layoutStore.getState());
+  React.useEffect(
+    () => layoutStore.subscribe((state) => (stateRef.current = state)),
+    [layoutStore]
+  );
+  const popoutContainer = usePopoutsStore((state) => {
+    if (!tabId) return undefined;
+    const tabLocation = getTabLocation(stateRef.current, tabId);
+    if (!tabLocation) return undefined;
+    if (!isPopoutTabLocation(tabLocation)) return undefined;
+    return state.popouts[tabLocation.popoutWidgetId] as HTMLElement;
+  });
   return (
-    <ScrollableWidgetContent itemId={itemId} providerId={providerId}>
-      <ErrorBoundary FallbackComponent={WidgetFallback}>
-        {widget?.reactNode}
-      </ErrorBoundary>
-    </ScrollableWidgetContent>
+    // Theme providers are required to open floating/popover elements in a popout widget window (instead of a main window).
+    <ThemeProvider
+      portalContainer={popoutContainer ?? undefined}
+      className="uifw-widgetPanels-content_themeProvider"
+    >
+      <ThemeProviderV2
```

Comment:
There is an issue with this: https://github.com/iTwin/appui/issues/612
Should we remove `ThemeProviderV2` and add a prereq: compatible with iTwinUI v3 only? Looking around if there's another option..

Reply:
Would that mean that the new popout widgets would only work with iTwinUI v3? But old ones would work fine with v2?

Reply:
Just FYI I had some success, simply wrapping each widget content in a v2 ThemeProvider, however, that involved re-parenting of ThemeProvider to the child window and would probably lead to other issues...

Reply:
Sounds good. Anyway, how long do we need to support iTwinUI v2 for?